src/java.base/share/classes/java/security/BasicPermission.java

```diff
@@ -217,12 +217,10 @@ public boolean equals(Object obj) {
 
 
     /**
-     * Returns the hash code value for this object.
+     * {@return the hash code value for this object}
```

Comment:
Where is this new javadoc syntax documented/specified? I recall seeing something about it, but not sure where and now I would like to read up on it.

Reply:
https://docs.oracle.com/en/java/javase/20/docs/specs/javadoc/doc-comment-spec.html#return